crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1673,7 +1685,7 @@ impl<'db> Type<'db> {
                 (Some(KnownClass::VersionInfo), "minor") => {
                     Type::IntLiteral(Program::get(db).python_version(db).minor.into()).into()
                 }
-                _ => class.instance_member(db, name),
+                _ => class.instance_member(db, name).0,
```

Comment:
Nit: adding a method would help with readability here (I think it's the type but hard to tell)

Reply:
agreed -- or it could just be a struct with named fields rather than a tuple struct?